publishable/config/voyager.php

```diff
@@ -94,21 +95,19 @@
     */
 
     'widgets' => [
-        [
-            'name'  => 'User',
+        'user' => [
             'icon'  => 'voyager-group',
             'model' => TCG\Voyager\Models\User::class,
             'url'   => 'admin/users',
             'image' => '/images/widget-backgrounds/02.png',
         ],
```

Comment:
I agree with you there. Maybe we should remake the entire configuration for the `widgets` group instead.
Thoughts?